next/app/about/leadership/team.ts

```diff
@@ -0,0 +1,127 @@
+interface TeamMember {
+  avatar: string;
+  name: string;
+  title: string;
+  desc: string;
+  linkedin?: string; 
+  github?: string;
+  email?: string;
+}
+
+export interface Team {
+  primary_officers: TeamMember[];
+  committee_heads: TeamMember[];
+}
+
+export const teamData = {
+    primary_officers: [
+      {
+        avatar: "https://randomuser.me/api/portraits/lego/2.jpg",
+        name: "Jonathan Cruz",
+        title: "President",
+        desc: "Lorem Ipsum is simply dummy text of the printing and typesettin industry.",
+        linkedin: "javascript:void(0)",
+        github: "javascript:void(0)",
+        email: "javascript:void(0)",
+      },
+      {
+        avatar: "https://randomuser.me/api/portraits/lego/3.jpg",
+        name: "Dominique Smith-Rodriguez",
+        title: "Vice President",
+        desc: "Lorem Ipsum is simply dummy text of the printing and typesettin industry.",
+        linkedin: "javascript:void(0)",
+        github: "javascript:void(0)",
+        email: "javascript:void(0)",
+      },
+      {
+        avatar: "https://randomuser.me/api/portraits/lego/4.jpg",
+        name: "Brendan Young",
+        title: "Treasurer",
+        desc: "Lorem Ipsum is simply dummy text of the printing and typesettin industry.",
+        linkedin: "javascript:void(0)",
+        github: "javascript:void(0)",
+        email: "javascript:void(0)",
+      },
+      {
+        avatar: "https://randomuser.me/api/portraits/lego/1.jpg",
+        name: "Fabi Marrufo Lopez",
+        title: "Secretary",
+        desc: "Lorem Ipsum is simply dummy text of the printing and typesettin industry.",
+        linkedin: "javascript:void(0)",
+        github: "javascript:void(0)",
+        email: "javascript:void(0)",
+      },
+    ],
+    committee_heads: [
```

Comment:
You used the order in the existing website, which makes a lot of sense. The order in which we display the officers is something to think about, especially as the API is starting to be built out. **Since this is all mock data, no need to spend time re-ordering it here.**

The API endpoint to get all active officers could return the list in some default sort order, whether that be alphabetical by title or some other method. @joebaillie21